controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -576,6 +571,7 @@ bool master_thread::autoconfig_wsc_add_m2(std::shared_ptr<ieee1905_1::tlvWscM1>
     std::fill(config_data.network_key_attr().data,
               config_data.network_key_attr().data + config_data.network_key_attr().data_length,
               0xaa); //DUMMY
+    config_data.vendor_extensions_attr().subelement_value = WSC::TEARDOWN;
```

Comment:
That's already the default value, and it will have to be changed anyway as part of #227, so don't bother setting it here.